pymedphys/tests/experimental/pseudonymisation/test_pseudonymisation.py

```diff
@@ -41,6 +42,47 @@ def test_pseudonymise_file():
         )
 
 
+def _assert_values_changed_and_not_hardcoded(test_file_path, pseudonymised_file_path):
+    ds_input = pydicom.dcmread(test_file_path, force=True)
+    ds_pseudo = pydicom.dcmread(pseudonymised_file_path, force=True)
+    # simplistic stand-in to make sure *something* is happening
+    assert ds_input["PatientID"].value != ds_pseudo["PatientID"].value
+    # make sure that we are not accidentally using the hardcode replacement approach
+    assert ds_pseudo["PatientID"].value not in ["", "Anonymous"]
```

Comment:
I think this might be enough. Part of me wants more things checked, the other part of me is saying as long as a few things are checked this is okay. I'm divided. I'll leave it up to you :).

Reply:
The alternative is high overhead.  And doing that would only ensure the correct strategy is being used, which is getting tested elsewhere.

Reply:
Yup, awesome.